tests/integration/firebolt.dbtspec

```diff
@@ -6,8 +6,10 @@ target:
   engine_name: "{{ env_var('ENGINE_NAME') }}"
   user: "{{ env_var('USER_NAME') }}"
   password: "{{ env_var('PASSWORD') }}"
+  schema: "{{ env_var('SCHEMA') }}"
```

Comment:
Should we just hardcode this value?

Reply:
No, this is either a by-team or by-user setting, so it needs to be set locally. Since we don't have schemas, this string just gets appended to each table name, so people aren't overwriting each others' tables. As an example, my "schema" is "emf," so all my table names are prefixed with "emf_"

Reply:
Ach, sorry. Yes, it should be hard-coded. I was thinking of `profiles.yml`, not this integration test file.